tests/lib/rules/no-invalid-html-attribute.js

```diff
@@ -537,6 +560,74 @@ ruleTester.run('no-invalid-html-attribute', rule, {
         },
       ],
     },
+    {
+      code: '<link rel="shortcut"></link>',
+      output: '<link rel="shortcut icon"></link>',
+      errors: [
+        {
+          messageId: 'notAlone',
+          data: {
+            reportingValue: 'shortcut',
+            missingValue: 'icon',
+          },
+        },
+      ],
+    },
+    {
+      code: '<link rel="shortcut foo"></link>',
+      output: '<link rel="shortcut icon "></link>',
+      errors: [
+        {
+          messageId: 'neverValid',
+          data: {
+            reportingValue: 'foo',
+            attributeName: 'rel',
+          },
+        },
+        {
+          messageId: 'notPaired',
+          data: {
+            reportingValue: 'shortcut',
+            secondValue: 'foo',
+            missingValue: 'icon',
+          },
+        },
+      ],
+    },
+    {
+      code: '<link rel="shortcut  icon"></link>',
+      output: '<link rel="shortcut icon"></link>',
+      errors: [
+        {
+          messageId: 'spaceDelimited',
+          data: { attributeName: 'rel' },
+        },
+      ],
+    },
+    {
+      code: '<link rel="shortcut  foo"></link>',
+      output: '<link rel="shortcut icon  "></link>',
```

Comment:
```suggestion
      code: '<link rel="shortcut  foo"></link>',
      output: '<link rel="shortcut foo"></link>',
```
this is a fine autofix, but the test case should still be invalid after it's done.

Reply:
4de38ef